src/algebra/big_operators/intervals.lean

```diff
@@ -38,6 +39,19 @@ lemma prod_Ico_add [ordered_cancel_add_comm_monoid α] [has_exists_add_of_le α]
   (∏ x in Ico a b, f (c + x)) = (∏ x in Ico (a + c) (b + c), f x) :=
 @sum_Ico_add _ (additive β) _ _ _ _ f a b c
 
+lemma sum_neg_one_pow_eq_ite [ring β] {n : ℕ} :
+  ∑ i in range n, (-1 : β) ^ i = if even n then 0 else 1 :=
```

Comment:
To me writing this as something like
```suggestion
  ∑ i in range n, (-1 : β) ^ i = (n % 2 : ℕ) :=
```
looks more convenient, does it work well in your intended application?

Reply:
Yes. That would be fine as well. I can switch to this.